holepunchto/pear-docs
# Add Inspector tab

Adds inspect feature. Works together with https://github.com/holepunchto/pear/pull/411.

<img width="1090" alt="image" src="https://github.com/holepunchto/pear-pulse/assets/376654/7770b59a-419e-4ab2-bce8-a54faabbb203">

How to test:

1. Use https://github.com/holepunchto/pear/pull/411
2. Create Pear terminal app (`{"pear": {"type": "terminal"}}` in the app's `package.json`)
3. Run the app like `$ pear.dev dev . --inspect`. The app will output a 64-string long key
4. Paste the key into the field in the Inspector tab in Pear Pulse
5. Open `chrome://inspect` and see your app there
